cookiecutter.json

```diff
@@ -1,5 +1,5 @@
 {
-    "project_slug": "sample",
+    "project_slug": "new-sample",
```

Comment:
Why is this change needed? Doesn't it get overwritten by our scaffold generator?

Reply:
Changed this to not overwrite my old setup. I'll change it back.